projects.php

```diff
@@ -2,42 +2,43 @@
 require_once './database.php';
 
 //get projects
-$sql = "SELECT * FROM `project`;";
+$sql = "SELECT `id`, `title`, `description`, `site_url`, `code_url` FROM `project`;";
 $query = $db->prepare($sql);
 $query->execute();
 $projects = $query->fetchAll();
 
 //get tags
-$sql = "SELECT * FROM `tag`;";
+$sql = "SELECT `id`, `name` FROM `tag`;";
 $query = $db->prepare($sql);
 $query->execute();
 $tags = $query->fetchAll();
 
 //get project tag relationships
-$sql = "SELECT *, `tag`.`name` as name FROM `project_tag`
+$sql = "SELECT `project_id`, `tag`.`name` as tagname FROM `project_tag`
         JOIN `tag` ON `tag_id`=`tag`.`id`;";
 $query = $db->prepare($sql);
 $query->execute();
 $project_tags = $query->fetchAll();
```

Comment:
to get the project tags for each project you should loop the projects after getting them, and call a function that gets all the tags for that project. this will be tidier than using the if with array_push you currently have. come speak to me if you dont understand.